runelite-client/src/main/java/net/runelite/client/plugins/banlist/BanListPlugin.java

```diff
@@ -121,22 +122,19 @@ protected void shutDown() throws Exception
 	@Subscribe
 	public void onConfigChanged(ConfigChanged event)
 	{
-		if (event.getGroup().equals("banlist"))
+		if (event.getGroup().equals("banlist") && event.getKey().equals("bannedPlayers"))
 		{
-			if (event.getKey().equals("bannedPlayers"))
+			for (String manual : Text.fromCSV(config.getBannedPlayers()))
```

Comment:
Won't this cause any issues if the string is empty? Also keep in mind you're doing contains before standardizing, so it would say no, doesn't contain while it actually may